chapter_09_docker.asciidoc

```diff
@@ -1417,7 +1420,7 @@ and the `target` path _inside_ the container:
 ----
 $ *docker build -t superlists . && docker run \
   -p 8888:8888 \
-  --mount type=bind,source=./src/db.sqlite3,target=/src/db.sqlite3 \
+  --mount type=bind,source=${PWD}/src/db.sqlite3,target=/src/db.sqlite3 \
```

Comment:
does the `.` not work?

Reply:
The `.` works in current version of docker. I'll revert this proposed edit.

The `.` fails in earlier versions of docker. And I did not try to upgrade docker before I proposed this edit.

## Extra details
### Where the `.` does not work for me
```bash
$ docker --version
Docker version 20.10.14, build a224086
```
and
```
macOS version 11.7.10
```

On that mac I have not found a way to upgrade the docker engine to a more recent version. 
I suspect I won't be able to upgrade the docker engine on that device (see screenshot below).
`${PWD}` works as a replacement for `.` in that setting.

### `.` works in more recent versions of docker
On a separate device where I have
```
macOS Version 12.7.4
```
and 
```
$ docker --version
Docker version 20.10.8, build 3967b7d
```
the `.` failed. But on that machine I was able to upgrade the docker engine
```
$ docker --version
Docker version 26.1.1, build 4cf5afa
```
And after the upgrade the `.` works.

On a third device with Ubuntu and that version of Docker (`26.1.1`), the `.` works.

Reply:
Screenshot for users of older version of macOS

![20240508-docker-desktop-for-macos-supports-latest-two-version-of-macos](https://github.com/hjwp/Book-TDD-Web-Dev-Python/assets/47086307/239935b2-ecac-439b-94f3-2d3df71994c2)

That refers to Docker Desktop and not necessarily the docker engine Docker Desktop uses. But I have not found a way to upgrade the engine independently of Docker Desktop. Savvier users may find a way. 

`${PWD}` instead of `.` is a workaround for users stuck with an old version of docker.